src/app/components/application/program-info/program-info.component.html

```diff
@@ -77,7 +77,7 @@ <h4>{{program.name}}  {{program.selectedInstrument}}</h4>
     <div *ngIf="program.isPrivateLesson" class="col-9 col-md-8 description">
       <h4 style="display: inline">{{program.name}}  {{program.selectedInstrument}}</h4>
       <span style="float: right" *ngIf="program.isRegistered && program.lessonCountAdd < 1"> {{program.lessonCount}} Lessons <a href="javascript:void(0);" (click)="addLessons(program, lessonsModal, 'registered')"> Add Lessons</a></span>
-      <span style="float: right" *ngIf="program.isRegistered && program.lessonCountAdd > 0"> {{program.lessonCount}} Lessons <span class="reviewError">+{{program.lessonCountAdd}} pending</span> <a href="javascript:void(0);" (click)="addLessons(program, lessonsModal, 'registered')"> Update Pending</a></span>
+      <span style="float: left" *ngIf="program.isRegistered && program.lessonCountAdd > 0"> {{program.lessonCount}} Lessons <span class="reviewError">+{{program.lessonCountAdd}} pending</span> <a href="javascript:void(0);" (click)="addLessons(program, lessonsModal, 'registered')" class="update-pending"> Update Pending</a></span>
```

Comment:
Feels like changing the float on this to `left` will mess up the alignment of the text while in non-mobile displays?

Reply:
![image](https://user-images.githubusercontent.com/125902283/225434762-81815455-36d5-4c68-87d7-cdd3637792d7.png)
This is in web so it works fine

Reply:
what does it look like with `float:right`?

Reply:
It looks the same to me on this scale but when you scale out, the 2 lesson phrase is not under cello with float right. With float left it sticks under cello

Reply:
I see so should I keep float right or change it to left? Feels consistent with other program cards

Reply:
My expectation is that they had it float right for a reason, maybe to draw extra attention to the # of lessons and pending lessons. Should maybe check in with @gugnitzja, Michael, and Margaret.

Reply:
multiple bootstrap float classes, actually to handle the case where ipad needs a different size

Reply:
I'd say float it left and we can discuss in UAT. I'm assuming it's easy enough to change later